autoscale-api/src/main/java/com/sequenceiq/periscope/api/endpoint/v1/AlertEndpoint.java

```diff
@@ -37,7 +37,7 @@
 
 @Path("/v1/clusters/{clusterId}/alerts")
 @Consumes(MediaType.APPLICATION_JSON)
-@Api(value = "/v1/alerts", description = ALERT_DESCRIPTION, protocols = "http,https")
+@Api(value = "/v1/alerts", description = ALERT_DESCRIPTION, protocols = "http,https", consumes = JSON)
```

Comment:
This doesn't change the API. It only needed to generate the same json that Spring would generate (with the help of the above `@Consumes` annotation).

Reply:
this should be consumes = MediaType.APPLICATION_JSON just to be consistent

Reply:
Yes I agree. I saw other examples as well where we had other constants like this which is already defined in the `MediaType`. Shall I remove them?